android-library/src/main/java/com/zetapush/library/Client.kt

```diff
@@ -0,0 +1,130 @@
+package com.zetapush.library
+
+import android.content.Context
+import com.zetapush.client.highlevel.ZetapushClient
+
+open class Client {
+
+    // Variables
+    protected var zetaPushService: ZetaPushService
+
+    /**
+     * Return the ZetaPush client
+     *
+     * @return
+     */
+    val zetaPushClient: ZetapushClient?
+        get() = zetaPushService.zetaPushClient
+
+    /**
+     * Get the user key of an user
+     *
+     * @return : User key as a string
+     */
+    val userKey: String?
+        get() = zetaPushService.userKey
+
+
+    /**
+     * Check if the user is connected to the ZetaPush platform
+     *
+     * @return : true if connected, false if not
+     */
+    val isConnected: Boolean
+        get() = zetaPushService.isConnected
+
+    /**
+     * Get the Sandbox ID
+     *
+     * @return : sandbox id as string
+     */
+    val sandboxId: String?
+        get() = zetaPushService.sandboxId
+
+    /**
+     * Get the resource
+     *
+     * @return : Resource as string
+     */
+    /**
+     * Set the resource
+     *
+     * @param resource : new resource
+     */
+    var resource: String?
+        get() = zetaPushService.resource
+        set(resource) {
+            zetaPushService.resource = resource
+        }
+
+    /**
+     * Constructor for a ZetaPush Client
+     */
+    constructor(context: Context) {
+        val storageTokenHandler = KeyValueTokenStorage(context, "key_storage_token")
```

Comment:
Can you just apply one of these constructors below ?
like : 
```
constructor(context: Context) {
  val storageCredentialHandler = KeyValueCredentialsStorage(context, "key_storage_login", "key_storage_password")
  this(context, storageCredentialHandler)
}
```

Reply:
in the next PR you will see this improvement a lot more convenient than this